source/MaterialXRuntime/Private/PvtApi.cpp

```diff
@@ -29,6 +29,21 @@ void PvtApi::reset()
     _unitDefinitions = UnitConverterRegistry::create();
 }
 
+void PvtApi::createLibrary(const RtToken& name)
```

Comment:
Create library function, note there is some code duplication here, if you have any recommendations, I'm all ears.